whistleblower/helpers/crop.py

```diff
@@ -0,0 +1,92 @@
+import sys
+
+import cv2
+import numpy
+
+TEXT_MIN_WIDTH = 35
+TEXT_MIN_HEIGHT = 10
+
+DEFAULT_WIDTH  = 850
+DEFAULT_HEIGHT = 1100
+
+KERNEL_WIDTH  = 25
+KERNEL_HEIGHT = 15
```

Comment:
What are the meaning of these constants?

Reply:
In theory, these are the distance (in pixels) between letters in the document to be considered the same line. They are used in the mathematical operations which glue the letters of the same line together. I'll add these comments.